content/developer/reference/backend/views.rst

```diff
@@ -1722,6 +1722,33 @@ Possible children of the view element are:
     Enables the 'quick create' and 'drag and drop' features when the kanban
     view is grouped by that field. Default: false.
 
+``header``
+  defines custom :ref:`buttons <reference/views/list/button>` in the control panel that perform an action/call a model's method.
```

Comment:
ref to **list** button ?

Reply:
it s the same api. We want to duplicate de text ?

Reply:
there's no button section in kanban?